aviary/backend/server/models.py

```diff
@@ -508,7 +508,7 @@ class AppArgs(BaseModel):
 
 
 class RouterArgs(BaseModel):
-    models: Dict[str, Union[str, LLMApp]]
+    models: Union[str, LLMApp, List[Union[LLMApp, str]]]
```

Comment:
This new typing doesn't support `Dict[str, Union[str, LLMApp]]` (the current typing). @Yard1 should we support that type?

Reply:
We don't have to anymore, no